test/sandbox-test.js

```diff
@@ -597,6 +597,7 @@ describe("sinonSandbox", function () {
 
         it("allows stubbing setters", function () {
             var object = {
+                foo: undefined,
```

Comment:
It is necessary, otherwise we run into the [input validation of collection.stub, that checks for own properties on stubbing targets](https://github.com/sinonjs/sinon/blob/5eca462a5f769c7de1622886be66d76a67f245a1/lib/sinon/collection.js#L93-L95).

`TypeError: Cannot stub non-existent own property foo`